.circleci/config.yml

```diff
@@ -204,6 +212,9 @@ jobs:
           compat: true
       - run_karma:
           force_native_shadow_mode: true
+      - run_karma:
+          disable_synthetic: true
+          enable_native_custom_element_lifecycle: true
```

Comment:
Is this missing an entry for `enabling native custom elements lifecycle: true` + `disable_synthetic: false`.

Reply:
I was not planning to test this feature with synthetic shadow DOM. My intention is that we would only use it in environments where synthetic shadow DOM is not loaded (e.g. LWR).

That said, in the future, we could definitely try to test in more combinations and make it support synthetic shadow. And I just tested it, and the tests do pass with synthetic shadow enabled.

Reply:
You know what, though, we might as well run the test. Added it! 🙂